frontend/src/bundles/auth/components/auth-layout/auth-layout.tsx

```diff
@@ -12,23 +12,37 @@ const AuthLayout: React.FC<Properties> = ({ children }) => {
         <>
             <Grid container className={styles.container}>
                 <Grid item xs={12} md={6}>
-                    <Grid item className={styles['selling-point']}>
+                    <Grid item className={styles.sellingPoint}>
                         <div className={styles.logo}></div>{' '}
                         {/* TODO: Change to Logo component*/}
```

Comment:
It can be out of bound of task, but logo already in development branch and can be added.

Reply:
Logo for this page will be another task. This task is for animation.

Reply:
Logo incorporated in this PR: https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/issues/276